deployment/network-operator/templates/mellanox.com_v1alpha1_nicclusterpolicy_cr.yaml

```diff
@@ -48,7 +48,11 @@ spec:
           {
             "resourceName": {{ $element.name | quote }},
             "rdmaHcaMax": 1000,
-            "devices": {{ $element.devices | toJson }}
+            "selectors": {
+              "vendors": {{ $element.vendors | toJson }},
```

Comment:
what happens if some attributes are missing  ? will it be empty ?

Reply:
Updated to be empty list

Reply:
updated